Automattic/wp-calypso
# Hosting Command Palette: Remove send feedback command

Related to https://github.com/Automattic/dotcom-forge/issues/6376

## Proposed Changes

This PR removes the `Send feedback` command from Calypso and wp-admin and ensures that when you type `feedback`, you are matched with `Get help` command.

## Testing Instructions



**To test Calypso**:

* Navigate to the Calypso Live link or pull the changes from this branch locally
* Navigate to `/sites`
* Press `cmd+k` on MacOs or `ctrl+k` on Windows to open the command palette
* Search for the `Send feedback` command
* Confirm that no command that is named `Send feedback` appears
* Confirm that the only command that gets matched to `send feedback` is `Get help`
* Confirm that `Get help` command still opens the Help Center correctly

**To test wp-admin**

* Run `install-plugin.sh command-palette-wp-admin
fix/remove-send-feedback-command` in your sandbox
* Sandbox `widgets.wp.com`
* Visit wp-admin of a Simple site or an Atomic site
* Press `cmd+k` on MacOs or `ctrl+k` on Windows to open the command palette
* Search for the `Send feedback` command
* Confirm that no command that is named `Send feedback` appears
* Confirm that the only command that gets matched to `send feedback` is `Get help`
* Confirm that `Get help` command still opens the Help Center correctly

## Pre-merge Checklist
